config/prod.webpack.config.js

```diff
@@ -23,10 +23,6 @@ plugins.push(
           __dirname,
           '../src/PresentationalComponents/ReportDetails'
         ),
-        './AdvisorRecommendationDetails': resolve(
```

Comment:
If I am not mistaken this page is consumed by content-preview application, we should make sure that we have communicated with them before removing.